simple-chat-back/src/models/messageModel.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const messageSchema = new mongoose.Schema({
+
+	user:{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User',
+        required: true,
+        trim: true
+    },
+    text:{
+        type: String,
+        required: true,
+        trim: true
+        
+    }
+},
+    {timestamps: true}
+)
+
+const Message = mongoose.model('Message', messageSchema )
```

Comment:
remove space between las parentheses (prettier would also automatically fix it for you)